plugins/airtable/src/NoAccess.tsx

```diff
@@ -1,6 +1,7 @@
 import { framer } from "framer-plugin"
 import { useLayoutEffect, useState } from "react"
 import auth from "./auth"
+import { showAccessErrorUI as showNoTableAccessUI } from "./ui"
```

Comment:
Why not use the original name? This function is only used in one place.

```suggestion
import { showAccessErrorUI } from "./ui"
```

Reply:
Failed rename - will fix!